sources/editor/Stride.Core.Assets.Editor/ViewModel/DirectoryBaseViewModel.cs

```diff
@@ -226,11 +222,6 @@ bool IAddChildViewModel.CanAddChildren(IReadOnlyCollection<object> children, Add
                         message = "Incompatible folder";
                         return false;
                     }
-                    if (directory.Root is ProjectViewModel && Root != directory.Root)
```

Comment:
I can't see an obvious reason why this (and the below) is being deleted?

Reply:
Ah, I should have put a comment for this one since it's not obvious.   The compiler identified that directory.Root can never be ProjectViewModel and was issuing a warning.

Reply:
Unclear if you are suggesting a change here or not.

Reply:
No, keep it removed